criteria/open-to-contributions.md

```diff
@@ -2,51 +2,43 @@
 order: 4
 ---
 
-# Welcome contributions
+# Welcome contributors
 
 ## Requirements
 
-* The codebase MUST have a public issue tracker that accepts suggestions from anyone.
 * The codebase MUST allow anyone to submit suggestions for changes to the codebase.
-* The documentation MUST link to both the public issue tracker and submitted codebase changes, for example in a `README` file.
-* The codebase MUST include an email address for security issues and responsible disclosure.
 * The codebase MUST include contribution guidelines explaining how contributors can get involved, for example in a `CONTRIBUTING` file.
-* The project MUST have communication channels for users and developers, for example email lists.
-* The codebase SHOULD have a publicly available roadmap.
-* The codebase SHOULD advertise the committed engagement of involved organizations in the development and maintenance.
-* The documentation SHOULD include instructions for how to report potentially security sensitive issues on a closed channel.
+* The codebase SHOULD advertize the committed engagement of involved organizations in the development and maintenance.
 * The codebase SHOULD document the governance of the codebase, contributions and its community, for example in a `GOVERNANCE` file.
+* The codebase SHOULD have a publicly available roadmap.
 * The codebase MAY include a code of conduct for contributors.
 
 ## Why this is important
 
-* Allows collaborative uptake of shared digital infrastructure.
-* Prevents forks of codebases, in which a community that works on a project splits because there is no shared progress.
+* Helps newcomers understand and trust your codebase community's leadership.
+* Prevents the community that works on a project splitting because there is no way to influence codebase goals and progress - resulting in diverging communities.
```

Comment:
I spot a hyphen that should be an en dash.

```suggestion
* Prevents the community that works on a project splitting because there is no way to influence codebase goals and progress – resulting in diverging communities.
```